src/biokbase/narrative/common/log_proxy.py

```diff
@@ -61,7 +61,7 @@ def __init__(self, input_file):
             if not isinstance(input_file, IOBase):
                 input_file = open(str(input_file), "r")
             try:
-                self._obj = yaml.load(input_file)
+                self._obj = yaml.load(input_file, Loader=yaml.SafeLoader)
```

Comment:
Another deprecation warning. Really, both `log_proxy` and `kblogging` aren't in use, I don't believe, but this was safe enough.